Source/ACE/Network/GameEvent/Events/GameEventPlayerDescription.cs

```diff
@@ -329,7 +304,12 @@ private void WriteEventBody()
 
             // FIXME(ddevec): We have duplicated data everywhere.  There is an AceObject CharacterOption flag, and a Player.CharacterOptions...
             //    Which one is right?  I have no idea.  Right now the aceObject works...  we should probably do a refactoring once we've restored functionality
-            var optionFlags = DescriptionOptionFlag.CharacterOption2;
+
+            // TODO: Refactor this to set all of these flags based on data. Og II
+            CharacterOptionDataFlag optionFlags = CharacterOptionDataFlag.CharacterOptions2;
+            if (aceObj.SpellsInSpellBars.Exists(x => x.AceObjectId == aceObj.AceObjectId))
```

Comment:
This is a special case and I think in this instance should be allowed.   The server never does anything with these.   It is only a persistence function.   It saves them and it reads them to load them.  Everything else happens client side in regards to spell bars.   I can easily add another layer - but it would be for nothing I believe.